### DISCUSSION TITLE
MT1011 recommendation

### DISCUSSION THREAD

**Discussion by robybrisson** [maester365/maester#490] [General]
The test's text indicates that security info registration should be done from trusted locations (makes sense).  The check seems to be returning "True" when trusted locations are instead excluded, instead allowing users to register only from non-trusted locations.

$result = $false
    foreach ($policy in $policies) {
        if ( $policy.conditions.users.includeUsers -eq "All" `
                -and $policy.conditions.clientAppTypes -eq "all" `
                -and $policy.conditions.applications.includeUserActions -eq "urn:user:registersecurityinfo" `
                -and $policy.conditions.users.excludeRoles -eq "62e90394-69f5-4237-9190-012177145e10" `
                -and $policy.conditions.locations.excludeLocations -eq "AllTrusted" `

Am I misunderstanding the intention of this policy?

Thanks,
Roby!

**Comment by l-gosling** [comment]
In the include should be all. @robybrisson 
The test logic changed a few weeks ago. I think now it makes more sense. 
https://github.com/maester365/maester/pull/1136/files
